cpp/scripts/check_extension_naming.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from __future__ import print_function
+import re
+import os
+import sys
+import subprocess
+import argparse
+
+
+
+DEFAULT_DIRS = ["cpp/bench",
+                "cpp/comms/mpi/include",
+                "cpp/comms/mpi/src",
+                "cpp/comms/std/include",
+                "cpp/comms/std/src",
+                "cpp/include",
+                "cpp/examples",
+                "cpp/src",
+                "cpp/src_prims",
+                "cpp/test"]
+
+IGNORE_FILES = r"(cpp/src/decisiontree/memory.h$|cpp/include/cuml/common/cubAllocatorAdapter.hpp$|cpp/include/cuml/datasets/make_arima.hpp$)"
```

Comment:
I think it's better to remove the arima component from this regex, as per my above comment.